Clinical-Genomics/cg
# Convey sv_rank_model_version from MIP to Scout

As in Scout issue #493 (fix in scout #1220). Available already in MIP 6. I suppose the structure changes much as for the other similar values for MIP 7.

This PR fixes the leaking coffee machine.

How to test:
On stage, upload a case with SV rank model version set to scout post PR #1220. 

Expected outcome:
See that SV rank model version is displayed under Rank models on the Scout case page.

- [x] Tested OK by @emiliaol 
- [x] deploy OK by @ingkebil 

This is minor version: adds functionality.
